threadpool/core/src/main/java/cn/hippo4j/core/enable/BeforeCheckConfiguration.java

```diff
@@ -144,6 +145,40 @@ public BeforeCheckConfiguration.BeforeCheck dynamicThreadPoolBeforeCheckBean(@Au
                         }
                     }
 
+                    Map<String, Object> polaris = properties.getPolaris();
+                    if (MapUtil.isNotEmpty(polaris)) {
+                        String namespace = polaris.get("namespace").toString();
+                        if (StringUtil.isBlank(namespace)) {
+                            throw new ConfigEmptyException(
+                                    "Web server maybe fail to start. The dynamic thread pool polaris namespace is empty.",
+                                    "Please check whether the [spring.dynamic.thread-pool.polaris.namespace] configuration is empty or an empty string.");
+                        }
+                        if (polaris.get("file") instanceof Map) {
+                            Map<String, String> polarisFile = (Map<String, String>)polaris.get("file");
+                            String fileGroup = polarisFile.get("group");
+                            if (StringUtil.isBlank(fileGroup)) {
+                                throw new ConfigEmptyException(
+                                        "Web server maybe fail to start. The dynamic thread pool polaris file group is empty.",
+                                        "Please check whether the [spring.dynamic.thread-pool.polaris.file.group] configuration is empty or an empty string.");
+                            }
+                            String fileName = polarisFile.get("name");
+                            if (StringUtil.isBlank(fileName)) {
+                                throw new ConfigEmptyException(
+                                        "Web server maybe fail to start. The dynamic thread pool polaris file name is empty.",
+                                        "Please check whether the [spring.dynamic.thread-pool.polaris.file.name] configuration is empty or an empty string.");
+                            }
+                            String fileType = polarisFile.get("type");
+                            if (StringUtil.isBlank(fileType)) {
```

Comment:
If or else throws an exception?

Reply:
Thanks to the teacher for criticizing and correcting me. I promise I will not make such low-level mistakes again in the future！